python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -180,13 +181,13 @@
 # posterior distribution, justifying the use of Monte-Carlo Markov chain
 # techniques, as described hereafter.
 
```

Comment:
please replace the graphical cell "Plot the simulated dataset" by the code below:
```python
# %%
# Plot the simulated dataset.
graph = ot.Graph("Simulated data", "$X_1$", "$Y$", True, "topleft", 16)
cloud_obs = ot.Cloud(X[:, 1].asPoint(), Yobs_sim)
cloud_obs.setPointStyle("bullet")
cloud_sim = ot.Cloud(X[:, 1].asPoint(), Y_sim)
cloud_sim.setPointStyle("bullet")
curve = ot.Curve(X[:, 1].asPoint(), mean_true)
curve.setLineWidth(1.5)
graph.add(curve)
graph.add(cloud_sim)
graph.add(cloud_obs)
graph.setLegends(["Trend", "$Y^{sim}$", "$Y^{obs}$"])
graph.setColors(ot.Drawable.BuildDefaultPalette(3))
_ = View(graph)
```